browser/ui/webui/ai_chat/ai_chat_ui_page_handler.cc

```diff
@@ -76,6 +81,17 @@ namespace ai_chat {
 
 namespace {
 
+class AssociatedLinkContentDelegate
+    : public ai_chat::AssociatedLinkContent::Delegate {
+ public:
+  AssociatedLinkContentDelegate() = default;
+  ~AssociatedLinkContentDelegate() override = default;
+  void AttachTabHelpers(content::WebContents* web_contents) override {
+    ephemeral_storage::EphemeralStorageTabHelper::FromWebContents(web_contents);
```

Comment:
I don't know if we should explicitly list them here, at least for now these will apply to any similar non-tab contents so maybe we should have a separate global method like `AttachTabHelpersForNonTabContents` or something and call that from the existing global `AttachTabHelpers`. Then we can also call it here and anyplace else we need them? I'd just like to keep the list someplace more visible